test/server/cards/01_SOR/events/MakeAnOpening.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('Make an Opening', function () {
+    integration(function () {
+        describe('Make an Opening\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['make-an-opening'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        groundArena: ['isb-agent'],
+                        spaceArena: ['cartel-spacer'],
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true }
+                    }
+                });
+            });
+
+            it('should reduce hp to a unit and heal your base', function () {
+                this.p1Base.damage = 5;
+                this.player1.clickCard(this.makeAnOpening);
+                expect(this.player1).toBeAbleToSelectExactly([this.pykeSentinel, this.isbAgent, this.cartelSpacer, this.sabineWren]);
+
+                this.player1.clickCard(this.isbAgent);
+                expect(this.isbAgent.getPower()).toBe(0);
```

Comment:
Sorry, missed this - can we select a unit that has power > 2 so we can confirm that the effect is reducing by exactly 2